### DISCUSSION TITLE
Improve Runner Instance information in logs

### DISCUSSION THREAD

**Discussion by TimPetricola** [runs-on/runs-on#47] [Ideas]
It's super useful to have all kind of info about the running information. Tiny details that could be improved:
- hide `SSH` line when ssh is disabled
- add CPU / RAM information (can be inferred from `InstanceType` but it's nice to have it easily accessible)
- add HDD size

**Comment by crohr** [comment]
This will land in https://github.com/runs-on/runs-on/pull/56

**Comment by crohr** [comment]
Implemented in v2.2.0
